modules/kmm-validation-status.adoc

```diff
@@ -6,43 +6,23 @@
 [id="kmm-validation-status_{context}"]
 = Validation status
 
-Preflight reports the status and progress of each module in the cluster that it attempts to
-validate.
-
-[source,terminal]
-----
-type CRStatus struct {
-	// Status of Module CR verification: true (verified), false (verification failed),
-	// error (error during verification process), unknown (verification has not started yet)
-	// +required
-	// +kubebuilder:validation:Required
-	// +kubebuilder:validation:Enum=True;False
-	VerificationStatus string `json:"verificationStatus"` <1>
-	// StatusReason contains a string describing the status source.
-	// +optional
-	StatusReason string `json:"statusReason,omitempty"` <2>
-	// Current stage of the verification process:
-	// image (image existence verification), build(build process verification)
-	// +required
-	// +kubebuilder:validation:Required
-	// +kubebuilder:validation:Enum=Image;Build;Sign;Requeued;Done
-	VerificationStage string `json:"verificationStage"` <3>
-	// LastTransitionTime is the last time the CR status transitioned from one status to another.
-	// This should be when the underlying status changed.  If that is not known, then using the time when the API field changed is acceptable.
-	// +required
-	// +kubebuilder:validation:Required
-	// +kubebuilder:validation:Type=string
-	// +kubebuilder:validation:Format=date-time
-	LastTransitionTime metav1.Time `json:"lastTransitionTime" protobuf:"bytes,4,opt,name=lastTransitionTime"` <4>
-}
-----
-
-The following fields apply to each module:
-
-<1> `VerificationStatus` - `true` or `false`, validated or not.
-
-<2> `StatusReason` - Verbal explanation regarding the status.
-
-<3> `VerificationStage` - Describes the validation stage being executed (Image, Build, Sign).
-
-<4> `LastTransitionTime` - The time of the last update to the status.
+A `PreflightValidation` resource reports the status and progress of each module in the cluster that it attempts or has attempted to validate in its `.status.modules` list. Elements of that list contain the following fields:
```

Comment:
Maybe worth updating to `PreflightValidationOCP` , to be consistent with the rest of the document.

Reply:
Dang! I missed that one.